<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="button_add_command">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Add to Bottom --&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="function_selector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>--Select Function--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sleep</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Carousel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Leadscrew</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Volume measurement</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="button_connect_serial">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="com_selector">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>--Select Port--</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="program_selector">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>--Select Program--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P04</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="value_selector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="baud_selector">
      <item>
       <property name="text">
        <string>--Select Baud Rate--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="button_write_program">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Write Program</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" rowspan="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>226</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="serial_status">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button_add_command_above">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Above Selection --&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="9">
     <widget class="QListWidget" name="program_list"/>
    </item>
    <item row="4" column="0" rowspan="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>147</width>
        <height>195</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="button_clear_list">
      <property name="text">
       <string>Clear List</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
